CHANGES.md

```diff
@@ -4,6 +4,10 @@ moodle-theme_boost_campus
 Changes
 -------
 
+### Unreleased
+
+* 2019-02-14 - Adding setting to show 'Add a block' widget in nav drawer as in theme Boost.
```

Comment:
I would not mention theme Boost here. Just use something like this:
"Setting to manage where the Add a block widget should be displayed."